docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -34,13 +35,21 @@ class Book(UUIDAuditBase):
 sqlalchemy_config = SQLAlchemyAsyncConfig(
     connection_string="sqlite+aiosqlite:///test.sqlite", session_config=session_config
 )  # Create 'async_session' dependency.
-sqlalchemy_plugin = SQLAlchemyInitPlugin(config=sqlalchemy_config)
+sqlalchemy_plugin = SQLAlchemyPlugin(config=sqlalchemy_config)
 
 
 async def on_startup() -> None:
-    """Initializes the database."""
+    """Initializes the database and add some dummy data."""
     async with sqlalchemy_config.get_engine().begin() as conn:
         await conn.run_sync(UUIDBase.metadata.create_all)
```

Comment:
This can be removed if we change change the `create_all=True` in the `SQLAlchemyConfig`
```suggestion
```